plenum/test/replica/test_buffers_cleaning.py

```diff
@@ -0,0 +1,63 @@
+from plenum.server.replica import Replica
+
+
+class FakeNode():
```

Comment:
This kind of test seems trivial to build with real nodes, why use a FakeNode then?